scss/bitstyles/ui/navbar.stories.mdx

```diff
@@ -33,8 +33,8 @@ Give the active link the `aria-current="page"` attribute.
   </ul>
 </details>
 
-<Canvas>
-  <Story name="Navbar">
+<Canvas >
+  <Story name="Navbar" inline={false}>
```

Comment:
The other stories all have an explicit height. Why does this one not?

Reply:
I missed it, thanks! 👍